app/scripts/models/auth_brokers/base.js

```diff
@@ -286,6 +286,10 @@ define([
        */
       emailVerificationMarketingSnippet: true,
       /**
+       * Should the view handle signed-in notifications from other tabs?
+       */
+      signedInNotificationHandler: true,
```

Comment:
I was hoping to avoid mentioning tabs because the dependency now is on the notifications object. Granted it is hard to envisage one of the other channels sending that notification, but the architecture doesn't prevent it. Worth considering or no?